src/sentry/static/sentry/app/utils/projects.jsx

```diff
@@ -151,21 +156,26 @@ class Projects extends React.Component {
    * results using search
    */
   loadAllProjects = async () => {
-    const {api, orgId, limit} = this.props;
+    const {api, orgId, limit, allProjects} = this.props;
 
     this.setState({
       fetching: true,
     });
 
     try {
-      const {results, hasMore} = await fetchProjects(api, orgId, {limit});
+      const {results, hasMore} = await fetchProjects(api, orgId, {limit, allProjects});
 
       this.setState({
         fetching: false,
         fetchedProjects: results,
         initiallyLoaded: true,
         hasMore,
       });
+
+      // populate the projects store if all projects were fetched
+      if (allProjects) {
+        ProjectActions.loadProjects(results);
```

Comment:
Should this be moved to `fetchProjects`?

Reply:
yup sounds good